sdk/websites/Azure.ResourceManager.AppService/src/GlobalSuppresions.cs

```diff
@@ -0,0 +1,5 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Azure.MgmtSdk.Analyzers", "AZM0020", Justification = "Already GA",
```

Comment:
We can suppress analyzer warnings here. The changes should be reviewed and approved.